server/src/main/java/org/apache/uniffle/server/ShuffleFlushManager.java

```diff
@@ -359,4 +359,8 @@ public long getCreateTimeStamp() {
       return createTimeStamp;
     }
   }
+
+  public ShuffleServer getShuffleServer() {
```

Comment:
Em... maybe we can add a `distribution` field in ShuffleBuffer, `getShuffleServer` may expose too much information.

Reply:
Done.

Reply:
I think the distribution should be bound to the app, which managed in ShuffleTaskManager will be better

Reply:
It's better to managed by ShuffleTaskManager. But I wanna to avoid using `getShuffleServer`.

Reply:
OK. Could we expose a method of `getDistirbution()` in `ShuffleFlushManager` ?

Reply:
OK.

Reply:
Done